adal/src/main/java/com/microsoft/aad/adal/AuthenticationDialog.java

```diff
@@ -262,8 +259,8 @@ public boolean processInvalidUrl(WebView view, String url) {
         }
 
         @Override
-        public void prepareForBrokerResumeRequest() 
-        {
+        public void prepareForBrokerResumeRequest() {
+            // Method intentionally left blank
```

Comment:
ok, i will suppress it.

---
In reply to: [73024826](https://github.com/AzureAD/azure-activedirectory-library-for-android/pull/689#discussion_r73024826) [](ancestors = 73024826)